packages/core/core-flows/src/payment-collection/workflows/create-payment-session.ts

```diff
@@ -77,7 +77,26 @@ export const createPaymentSessionsWorkflow = createWorkflow(
   ): WorkflowResponse<PaymentSessionDTO> => {
     const paymentCollection = useRemoteQueryStep({
       entry_point: "payment_collection",
-      fields: ["id", "amount", "currency_code", "payment_sessions.*"],
+      fields: [
```

Comment:
suggestion: we could add tests in `integration-tests/modules/__tests__/payment/payment-session.workflows.spec.ts` just to ensure the data are correctly pass and we don't break it later, wdyt?